CHANGELOG.md

```diff
@@ -7,18 +7,131 @@ and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0
 
 ## [Unreleased]
 
-## [0.1.0] - 2021-01-12
+### Added
+
+- Add `unstake` function to allow unstaking a previously staked amount [#58]
+- Add `fetch_existing_nullifiers` to the `StateClient` [#41]
+
+### Changed
+
+- Change `withdraw` function to withdraw the reward for staking and
+  participating in the consensus [#58]
+- Change `stake` and `withdraw` cryptographic signatures to what the stake
+  contract expects in the new spec [#58]
+- Change `StakeInfo` to have an optional amount staked, a reward, and a
+  signature counter [#58]
+- Change signature of `fetch_notes` by removing block height
+- Delegate buffer allocation in FFI `fetch_notes` to the user. [#58]
```

Comment:
Not all the item list are in the form `Change/Update/etc`, this is okay for now but keep in mind that doing so we won't be able to generate the CHANGELOG from those commits.